angular/package.json

```diff
@@ -34,7 +34,7 @@
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "~2.0.0",
-    "ng-packagr": "^14.1.0",
-    "typescript": "~4.7.2"
+    "ng-packagr": "^15.0.3",
```

Comment:
@bdsl Shouldn't this be 15.0.4?

Reply:
No, the latest release is 15.0.3: https://www.npmjs.com/package/ng-packagr